src/uglify/serialization.js

```diff
@@ -0,0 +1,46 @@
+const toType = value => (Object.prototype.toString.call(value).slice(8, -1));
```

Comment:
I don't think we can legitimately do this for functions, though. 

```
var a = 1;
function toBeSerialized() {
  console.log(a);
}
```

in the case above, this results in non-obvious errors when serializing and deserializing in a different context. Is there any way we can avoid passing functions or regexes down to the worker and instead post process once comments are sent back to us?